scheduler-server/sql/scheduler/tables/scheduling_request.sql

```diff
@@ -10,6 +10,7 @@ create table scheduling_request (
   reason jsonb null,
   canceled boolean not null default false,
   dataset_id integer default null,
+  for_aerie_scheduler boolean not null default true,
```

Comment:
Proposal (ignorable):
```suggestion
  use_external_scheduler boolean not null default false,
```
```sql
comment on column scheduling_request.use_external_scheduler is e''
 'Whether the request will be (or has been) handled by an external scheduler.';
```